plugins/asterisk/asterisk

```diff
@@ -27,13 +30,15 @@ This plugin will produce multiple graphs showing:
 The following configuration parameters are used by this plugin
 
  [asterisk]
-  env.host     - hostname to connect to
-  env.port     - port number to connect to
-  env.username - username used for authentication
-  env.secret   - secret used for authentication
-  env.channels - The channel types to look for
-  env.codecsx  - List of codec IDs (hexadecimal values)
-  env.codecs   - List of codecs names, matching codecsx order
+  env.host       - hostname to connect to
+  env.port       - port number to connect to
+  env.username   - username used for authentication
+  env.secret     - secret used for authentication
+  env.channels   - The channel types to look for
+  env.codecsx    - List of codec IDs (hexadecimal values)
+  env.codecs     - List of codecs names, matching codecsx order
+  env.meetme     - Set to 1 to enable graphs for the MeetMe application
```

Comment:
Please use `enable_meetme` or `meetme_enabled` instead. This clarifies the content of the variable.
Same for `confbridge` below.